ixwebsocket/IXWebSocket.cpp

```diff
@@ -41,7 +41,8 @@ namespace ix
         , _pingIntervalSecs(kDefaultPingIntervalSecs)
     {
         _ws.setOnCloseCallback(
-            [this](uint16_t code, const std::string& reason, size_t wireSize, bool remote) {
```

Comment:
This minor formatting change stems from simply applying clang format with the projects `.clang-format` configuration.